### ISSUE TITLE
Deprecate or document/test SCENES_IN_ORDER

### ISSUE THREAD

**Issue by leotrs** [ManimCommunity/manim#74]
The function `get_scene_classes_from_module` in `extract_scene.py` checks for a module member called `SCENES_IN_ORDER`. It appears that the user is allowed to assign a list to this that specifies the order in which the scenes will be rendered.

1. Given that there are plans to render scenes in parallel, this would become deprecated.
2. If we continue to support rendering scenes in series for some purpose, I'm still not sure what is the purpose of determining the rendering order.
3. Even if we wanted to do so, it seems more natural that the rendering order should be the order in which scenes appear in the module, rather than setting a separate variable for the order.
4. In any case, if the `SCENES_IN_ORDER` variable will still be used, we should document it somewhere. If it won't be used, we should delete it. (Lines 124 and 125 in `extract_scene.py` are the only mentions to this variable in the code base.)
5. Finally, if we do choose to support it, we should write a test for it.

**Comment by XorUnison** [comment]
Although we should implement multiprocessing, I would add this as an option, as opposed to replacing the current linear render process for reasons I'll mention in the new issue.

As for removing this... I mean it is an overly specific use case, yes. I do struggle of any **good** reasons to keep it.
The vast majority of people will have enough CPU power to at least benefit from multiprocessing a bit, and a scene order just doesn't do a whole lot for people unless they're sitting anxiously on the edge of their seat for every single result.

Anyone have a better case to make for why to keep this?

**Comment by leotrs** [comment]
I don't hear anyone making arguments for keeping this :)
